### DISCUSSION TITLE
Intensities reported

### DISCUSSION THREAD

**Discussion by camoin** [vdemichev/DiaNN#99] [General]
Hi Vadim, 
Thanks for this great software. 
There are a few things not clear to me regarding the intensities reported by DIA-NN. 
What is the difference in intensity between the report.pr.matrix file and the report.unique.genes.matrix file? 
Same question between report.pg.matrix and report.gg.matrix?
Can you clarify the intensities that are given in these files. 
Thank you
All the best
Luc

**Comment by vdemichev** [comment]
Hi Luc,

These files contain normalised intensities for precursor ions, genes identified and quantified using proteotypic peptides, protein groups and gene groups, respectively. However these matrices are provided for quick analyses, e.g. directly in Excel, and are very stringently filtered for low FDR. Ourselves we work with the main report (report.tsv, the largest .tsv file) using R/Python.

Best wishes,

Vadim
